applications/protocols_serialization/client/snippets/logging/logging.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_LOG_FORWARDER_RPC=y
```

Comment:
Seems that this config is incomplete or name is confusing. In logging snippet I would expect enabled logging subsystem. And it seems that for logging we have to add both debug and logging snippet. Order of those might matter.

Reply:
I can rename the snippet to "logrpc" or someting. Didn't want to do it because we don't add "rpc" to other snippets that enable RPC groups, but I can change this in this case.

Reply:
Does logging have sense without debugging?

Reply:
Sure, the crash log feature can be tested without logger enabled on the client side so I didn't want needlessly couple these two.

Reply:
@tomaszkob89 From CI POV how will it be easier to organise this?

Reply:
Can't this be just a part of proj.conf? (As I understand we want to have it enabled even if debugging is disabled, so far it is a single flag so a separate snippet is unnecessary?)

Reply:
I'll keep the current approach until we talk with PMT on what we want to deliver.